core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -97,18 +84,17 @@ angular.module('oppia').directive('aboutPage', [
 
             WindowRef.nativeWindow.onhashchange = function() {
               const hashChange = window.location.hash.slice(1);
-              if (hashChange === ctrl.TAB_ID_FOUNDATION || (
-                hashChange === 'license')) {
-                activateTab(ctrl.TAB_ID_FOUNDATION);
-                // Ensure page goes to the license section
-                if (hashChange === 'license') {
-                  WindowRef.nativeWindow.location.reload(true);
-                }
-              } else if (hashChange === ctrl.TAB_ID_CREDITS) {
-                activateTab(ctrl.TAB_ID_CREDITS);
-              } else if (hashChange === ctrl.TAB_ID_ABOUT) {
-                activateTab(ctrl.TAB_ID_ABOUT);
+              if (hashChange === 'license') {
+                ctrl.activeTabName = ctrl.TAB_ID_FOUNDATION;
+                WindowRef.nativeWindow.location.reload(true);
+              } else if (ALLOWED_TABS.includes(hashChange)) {
+                ctrl.activeTabName = hashChange;
               }
+
+              // A click event needs to be trigger by hand because
+              // onhashchange event doesn't update ctrl.activeTabName in
+              // the html. Just after a click the template is updated.
+              angular.element(document.querySelector('body')).click();
```

Comment:
I don't think this is a good approach, I would suggest you create a separate issue for the page doesn't focus on the correct place on reload and get this PR merged without fixing that issue. (Let's fix that issue separately, considering this issue isn't introduced by this PR.)

(I'll approve this PR once you undo these changes!)

Reply:
Done @DubeySandeep, here's the issue: #9162